PURL-SPECIFICATION.rst

```diff
@@ -412,9 +411,10 @@ To parse a ``purl`` string in its components:
 
 - Split the ``remainder`` on '/'
 
-  - Discard any empty segment from that split
   - Percent-decode each segment
   - UTF-8-decode each segment if needed in your programming language
+  - Discard any segment that is empty, or equal to `.` or `..`
```

Comment:
Discarding dot and doubledot? Where comes this from?
Dots are allowed here, and for some (upcoming) purl types actually a need.
Please revisit this change.

Reply:
I removed it from #455.